src/Spryker/Glue/AppPaymentBackendApi/Plugin/GlueApplication/PaymentConfirmDisconnectionRequestValidatorPlugin.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+/**
+ * This file is part of the Spryker Suite.
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace Spryker\Glue\AppPaymentBackendApi\Plugin\GlueApplication;
+
+use Generated\Shared\Transfer\GlueErrorTransfer;
+use Generated\Shared\Transfer\GlueRequestTransfer;
+use Generated\Shared\Transfer\GlueRequestValidationTransfer;
+use Generated\Shared\Transfer\PaymentConditionsTransfer;
+use Generated\Shared\Transfer\PaymentCriteriaTransfer;
+use Spryker\Glue\AppKernel\Plugin\GlueApplication\AbstractConfirmDisconnectionRequestValidatorPlugin;
+use Spryker\Glue\AppPaymentBackendApi\AppPaymentBackendApiConfig;
+use Spryker\Glue\AppPaymentBackendApi\Mapper\Payment\GlueRequestPaymentMapper;
+use Spryker\Zed\AppPayment\Business\Payment\Status\PaymentStatus;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * @method \Spryker\Glue\AppPaymentBackendApi\AppPaymentBackendApiFactory getFactory()
+ */
+class PaymentConfirmDisconnectionRequestValidatorPlugin extends AbstractConfirmDisconnectionRequestValidatorPlugin
+{
+    protected function getLabelOk(): string
+    {
+        return $this->getFactory()->getTranslatorFacade()->trans('Ignore & Disconnect');
+    }
+
+    protected function getLabelCancel(): string
+    {
+        return $this->getFactory()->getTranslatorFacade()->trans('Cancel');
+    }
+
+    protected function validateDisconnectionRequest(GlueRequestTransfer $glueRequestTransfer): GlueRequestValidationTransfer
+    {
+        if (empty($glueRequestTransfer->getMeta()[GlueRequestPaymentMapper::HEADER_TENANT_IDENTIFIER])) {
+            return (new GlueRequestValidationTransfer())
+                ->setIsValid(false)
+                ->addError(
+                    (new GlueErrorTransfer())
+                        ->setCode(AppPaymentBackendApiConfig::ERROR_CODE_PAYMENT_DISCONNECTION_TENANT_IDENTIFIER_MISSING)
+                        ->setMessage(
+                            $this->getFactory()->getTranslatorFacade()->trans('Tenant identifier is missing.'),
+                        ),
+                );
+        }
```

Comment:
Can't we cover this in a central place for all requests? `\Spryker\Glue\GlueApplication\Validator\RequestValidatorInterface`